spec/features/staff/users_can_filter_activities_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.feature "Users can filter activities" do
+  context "when the user is signed in as a BEIS user" do
+    let(:user) { create(:beis_user) }
+    before { authenticate!(user: user) }
+
+    scenario "they see the organisation filter on the activities index" do
+      visit root_path
+      within "#navigation" do
+        click_on "Activities"
+      end
+
+      expect(page).to have_content I18n.t("filters.activity.title")
+      expect(page).to have_select "organisation_id"
+    end
+
+    scenario "they can filter the activities to an organisation" do
```

Comment:
This test feels a little magical and could perhaps be made more explicit.

Rather than building a project and letting FactoryBot create the organisation and another parent activity. The fact that there are 2 organisations now in play is hard to uncover.

Could we instead go out of a way a bit to make it clearer with:

```
organisation_of_interest = create(:delivery_partner_organisation)
programme = create(:programme_activity)
project = create(:project_activity, parent: programme, organisation: organisation_of_interest)

expect(page).to have my_programmes

select organisation_of_interest.name, from: "organisation_id"
click_on I18n.t("filters.activity.submit")

expect(page).to have their_projects
```